adapters/repos/db/helpers/helpers_test.go

```diff
@@ -0,0 +1,108 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2023 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package helpers
+
+import (
+	"bytes"
+	"encoding/binary"
+	"testing"
+
+	"github.com/weaviate/weaviate/adapters/repos/db/inverted/tracker"
+)
+
+func TestMakePropertyPrefix(t *testing.T) {
+	tracker, err := tracker.NewJsonPropertyIdTracker("/tmp/test.json")
+	if err != nil {
+		t.Fatalf("Expected no error, got %v", err)
+	}
+
+	_, err = tracker.CreateProperty("property")
+	if err != nil {
+		t.Fatalf("Expected no error, got %v", err)
+	}
+
+	prefix, err := MakePropertyPrefix("property", tracker)
```

Comment:
Please explain which part of this function requires us to create an object, a hash table and a logger so that we can concatenate two strings?

Reply:
True, concatenating string does not require state/struct. I proposed creating helper struct or attach those methods to tracker to have place to store logger instance, otherwise logger would have to be passed to function as an 3rd argument.
But since logging is removed there is no need to change anything.